### PR TITLE
[Rust] Audio demo

### PR DESCRIPTION
**Pull request by ondrejkolin**

### CODE REVIEW

File: src/Audio/code.rs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+use crate::workbench;
+use gtk::glib;
+use gtk::prelude::*;
+use std::collections::HashMap;
+
+pub fn main() {
+    //Otherwise the MediaFile crate crashes
```
Comment (Hofer-Julian) [inline src/Audio/code.rs:7]:
```suggestion
    // Otherwise the MediaFile crate crashes
```

File: src/Audio/code.rs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            .unwrap();
+
+        button.connect_clicked(glib::clone!(
+          @weak controls => move |_| {
+            if let Some(media_stream) = controls.media_stream() {
+                media_stream.set_playing(false);
+            }
+
+            controls.set_media_stream(Some(&gtk::MediaFile::for_file(
+              &gtk::gio::File::for_uri(&workbench::resolve(file_name))
```
Comment (Hofer-Julian) [inline src/Audio/code.rs:29]: Please always bring `glib` and `gio` into scope if you use them

**Review by Hofer-Julian — COMMENTED** [review]
Small nits, apart from that it looks good!

**Comment by ondrejkolin** [comment]
I think I've fixed this one ... I'll update the other ones too...

**Comment by ondrejkolin** [comment]
Thanks for the hints I've updated all the PRs

**Review by Hofer-Julian — APPROVED** [review]
Thanks!

**Pull request merged**